doc/modules/feature_extraction.rst

```diff
@@ -916,6 +916,30 @@ Some tips and tricks:
 
     (Note that this will not filter out punctuation.)
 
+
+    The following example will, for instance, transform British spelling to American
```

Comment:
Very nice example!

Another use for this, maybe more useful in practice, is normalizing all numbers to a new `##` token